app/services/physical_infra_topology_service.rb

```diff
@@ -69,7 +73,10 @@ def entity_status(entity)
 
   # Decide whether or not to add an entity to the graph stack
   def add_to_graph?(entity, links_index, links, parent_id)
-    return true if entity.kind_of?(Tag)
+    return true if entity.kind_of?(Tag) || entity.kind_of?(PhysicalSwitch)
+
+    # return parent_id.downcase.include?("physicalserver") if entity.kind_of?(PhysicalSwitch)
```

Comment:
What is this line and why is it commented?

Reply:
My bad, let me remove that